docs/integration-handbook.md

```diff
@@ -228,6 +228,10 @@ All vehicles will have a width of 100cm, a length of 180cm and a height of 50cm.
 This is the bounding box of the vehicle (the part of the vehicle that interacts with the world)
 The 3d models are not restricted to this size.
 
+Teams participating in FSOnline can provide their 3d vehicle models to be used during competition. 
+If you want the 3d model to be public, you can create a pr to this repo.
```

Comment:
```suggestion
If you want the 3d model to be public, you can create a pull request to this repository.
```
This might seem redundant, but I like to be explicit 😉